addons/sourcemod/scripting/tfgo/config.sp

```diff
@@ -59,14 +94,9 @@ void ReadKillAwardConfig(KeyValues kv)
 
 void Config_Init()
 {
-	if (g_WeaponClassKillAwards == null)
-		g_WeaponClassKillAwards = new StringMap();
-	
-	if (g_AvailableWeapons == null)
-	{
-		Weapon weapon;
-		g_AvailableWeapons = new ArrayList(1 + sizeof(weapon));
-	}
+	if (g_WeaponClassKillAwards == null) g_WeaponClassKillAwards = new StringMap();
+	if (g_AvailableWeapons == null) g_AvailableWeapons = new ArrayList(sizeof(Weapon) + 1);
+	if (g_AvailableGear == null) g_AvailableGear = new ArrayList(sizeof(Gear) + 1);
```

Comment:
`+ 1` shouldn't be needed when `sizeof` on enum struct gives correct required length